src/cljs/athens/electron.cljs

```diff
@@ -325,6 +325,52 @@
   ;; open or create a new starter db
 
   ;; Watch filesystem, e.g. in case db is updated via Dropbox sync
+  
+    ;; Update index.transit    
+  (defn write-db-index [file filepath]
+    (go   
```

Comment:
Why `go` is used here?

Reply:
Hi @neotyk , go enable the async mode i used go to avoid a loop while the file is being copied.

Reply:
I don't see anything that would justify using `go` block here.
You're calling `fs.writeFileSync` from js land, I see no `core.async` here.